bitnami/janusgraph/templates/configmap.yaml

```diff
@@ -0,0 +1,47 @@
+{{- /*
+Copyright VMware, Inc.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+apiVersion: v1
```

Comment:
Should we support `existingConfigmap`?

Reply:
Nice catch! We do support it, but I forgot the conditional to not render this.